api-tests/core/strapi/document-service/find-first.test.api.ts

```diff
@@ -18,58 +17,46 @@ describe('Document Service', () => {
   });
 
   describe('FindFirst', () => {
-    it(
-      'find first document with defaults',
-      testInTransaction(async () => {
-        const articleDb = await findArticleDb({ title: 'Article1-Draft-EN' });
-
-        const article = await strapi.documents(ARTICLE_UID).findFirst({});
-
-        expect(article).not.toBeNull();
-        expect(article).toMatchObject(articleDb);
-      })
-    );
-
-    it(
-      'find first document in french',
-      testInTransaction(async () => {
-        const articleDb = await findArticleDb({ title: 'Article1-Draft-FR' });
-
-        const article = await strapi.documents(ARTICLE_UID).findFirst({
-          locale: 'fr',
-          filters: {
-            title: { $startsWith: 'Article1' },
-          },
-        });
-
-        expect(article).toMatchObject(articleDb);
-      })
-    );
-
-    it(
-      'find one published document',
-      testInTransaction(async () => {
-        const article = await strapi.documents(ARTICLE_UID).findFirst({
-          status: 'published',
-        });
-
-        expect(article).toMatchObject({
-          publishedAt: expect.any(String),
-        });
-      })
-    );
-
-    it(
-      'find first draft document',
-      testInTransaction(async () => {
-        const article = await strapi.documents(ARTICLE_UID).findFirst({
-          status: 'draft',
-        });
-
-        expect(article).toMatchObject({
-          publishedAt: null,
-        });
-      })
-    );
+    it('find first document with defaults', async () => {
+      const articleDb = await findArticleDb({ title: 'Article1-Draft-EN' });
+
+      const article = await strapi.documents(ARTICLE_UID).findFirst({});
```

Comment:
Should we return the draft by default? I think this has been our approach until now, but I think we should return published data by default. wdyt?

Reply:
I also think published should be the default. Let's not update all the tests yet though until we confirm with Alex